emailtemplates/models.py

```diff
@@ -18,7 +18,14 @@ def get_template_choices(templates_dir_name='emailtemplates', suffix='.html'):
     returns list of html templates from emailtemplates/*/
     """
     templates_list = []
-    for template_dir_path in (settings.TEMPLATE_DIRS + get_app_template_dirs('templates')):
+
+    try:
+        template_dir = settings.TEMPLATE_DIRS
+    except AttributeError:
+        template_dir = settings.TEMPLATES[0]['DIRS']
```

Comment:
I'd add a comment about Django compatibility + iterate over TEMPLATES to make sure that not only the first backend will be checked. In this case it might make sense to rename template_dir -> template_dirs.